server/session/handler_item_stack_request.go

```diff
@@ -85,7 +91,7 @@ func (h *ItemStackRequestHandler) handleRequest(req protocol.ItemStackRequest, s
 			err = h.handleBeaconPayment(a, s)
 		case *protocol.CraftCreativeStackRequestAction:
 			err = h.handleCreativeCraft(a, s)
-		case *protocol.CraftResultsDeprecatedStackRequestAction:
+		case *protocol.ConsumeStackRequestAction, *protocol.CraftResultsDeprecatedStackRequestAction:
```

Comment:
I'm glad you're ignoring the ConsumeStackRequestAction, we probably shouldn't be using this. (Might even expose a vulnerability in vanilla?)